scratch/foo/src/Foo.scala

```diff
@@ -2,6 +2,8 @@ package foo
 object Foo{
 
   def main(args: Array[String]): Unit = {
-    println(1 == "")
+    println("enter any character: ")
+    val c = System.in.read().toChar
+    println("c: [" + c + "]")
```

Comment:
As we encourage users to use `scratch` to locally play around, we should not modify it upstream.